assets/style/_header.scss

```
@@ -1,28 +1,32 @@
 header {
-  height: 32.5rem;
+
+  h1 {
+    font-family: 'Prestafont';
```

```suggestion
    font-family: Prestafont, sans-serif;
```
And for other places too!

---

Fixed in https://github.com/PrestaShop/TopContributors/pull/118/commits/ccb67fff2bd015ae2c21aa81fdc391f713eaf909